README.md

```diff
@@ -1,33 +1,39 @@
----
-tags: HideReplier
----
-
```

Comment:
- for HackMD
- any better solution?

Reply:
You should push to a new branch from HackMD, and modify the markdown file to be compatible with GitHub, then merge into master. [Here](https://github.com/penut85420/HelloHackMD) is a demonstration repo.

Reply:
Got it! thx